cmd/off.go

```diff
@@ -48,5 +55,7 @@ func newOffCmd(h *pkg.Hctl, out io.Writer) *cobra.Command {
 		},
 	}
 
+	cmd.PersistentFlags().Float64Var(&transition, "transition", 0, "Set transition time in seconds")
```

Comment:
Keep the help text in sync for `--transition` in `on` and `off`. I'm fine with either of them (with `1.5` example or without), you decide.